nixos/modules/services/misc/klipper.nix

```diff
@@ -17,6 +17,26 @@ in
         description = "Allows Octoprint to control Klipper.";
       };
 
+      user = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = ''
+          User account under which Klipper runs.
+
+          If null is specified (default), a temporary user will be created by systemd.
+        '';
+      };
+
+      group = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = ''
+          Group account under which Klipper runs.
+
+          If null is specified (default), a temporary user will be created by systemd.
+        '';
+      };
+
```

Comment:
I think it's best to just default to a `klipper` DynamicUser.

Reply:
If it's `null` then it will use `DynamicUser` same as the current behavior.